source-zendesk-support/tests/test_snapshots.py

```diff
@@ -29,6 +29,41 @@ def test_capture(request, snapshot):
             unique_stream_lines.append(line)
             seen.add(stream)
 
+    streams_with_updated_at = [
+        "acmeCo/group_memberships",
+        "acmeCo/groups",
+        "acmeCo/macros",
+        "acmeCo/organizations",
+        "acmeCo/organization_memberships",
+        "acmeCo/posts",
+        "acmeCo/post_comments",
+        "acmeCo/post_comment_votes",
+        "acmeCo/post_votes",
+        "acmeCo/satisfaction_ratings",
+        "acmeCo/sla_policies",
+        "tacmeCo/icket_fields",
+        "acmeCo/ticket_metrics",
+        "acmeCo/ticket_skips",
+        "acmeCo/tickets",
+        "acmeCo/users",
+        "acmeCo/brands",
+        "acmeCo/custom_roles",
+        "acmeCo/schedules",
+        "acmeCo/ticket_forms",
+        "acmeCo/account_attributes",
+        "acmeCo/attribute_definitions"
+    ]
+
+    for l in unique_stream_lines:
+        type, rec = l[0], l[1]
+
+        rec['_meta']['row_id'] = 0
```

Comment:
It would probably be alright to simplify this along the lines of:

```
if "updated_at" in rec:
    rec['updated_at'] = 'redacted'
if "last_login_at" in rec:
    rec['last_login_at'] = 'redacted'
```

Reply:
That's a lot simpler, I'll make that change. Thanks for the suggestion!